src/pages/Search/EmptySearchView.tsx

```diff
@@ -258,6 +241,7 @@ function EmptySearchView({type, hasResults}: EmptySearchViewProps) {
                 title={content.title}
                 titleStyles={content.titleStyles}
                 subtitle={content.subtitle}
+                body={content.body}
```

Comment:
IMO it would be better to use `children` for this. It feels more "react"y. Would that be possible?

Reply:
ah yes, `children` is much much better